polaris-migrator/src/migrations/react-replace-text-components/react-replace-text-components.ts

```diff
@@ -8,6 +8,7 @@ import {replaceTextStyle} from './steps/replace-text-style';
 
 export interface MigrationOptions extends Options {
   relative: boolean;
+  componentNames?: string[];
```

Comment:
Unsure if this should be one or many. I made it many for now, but will likely run it one by one.